tests/Unit/inc/Engine/Optimization/RUCSS/Warmup/APIClient/sendWarmupRequest.php

```diff
@@ -53,10 +53,14 @@ public function testShouldSendWarmupRequest( $atts, $success, $mockResponse ): v
 		];
 
 		Functions\when( 'wp_parse_args' )->returnArg( 1 );
+
+		$api_url = rocket_get_constant( 'WP_ROCKET_SAAS_API_URL', false )
+			? rocket_get_constant( 'WP_ROCKET_SAAS_API_URL', false )
```

Comment:
This should use the Stub method available in `StubTrait` instead of using the function directly

Reply:
I think this works now.
Added a default return value for the new constant to the StubTrait.